### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by russbiggs** [osmfoundation/welcome-mat#19]
Since this a communication tool it might be good to improve the accessibility. This [lighthouse report](https://googlechrome.github.io/lighthouse/viewer/?gist=ccdf11453fc11b7c2e86f78c085a8912) shows the points that are failing. Some of these should be pretty straightforward fixes (labels, name, alt attributes etc.). Others like the color scheme probably require discussion. I'm happy to submit a PR to help improve some of this.

**Comment by woodpeck** [comment]
While there's nothing wrong per se with accessibility improvements, I am slightly worried about potential scope creep. What we are discussing here is *not* the new public-facing OpenStreetMap web site, not even the new public-facing OpenStreetMap Foundation website. What the "welcome mat" intends to do is explain to organisations (not so much to individuals) how the project works and what avenues are available for engaging with OpenStreetMap.

It would certainly be a waste of resources to try and make this "welcome mat" more accessible or more international than our existing, public-facing web sites openstreetmap.org and osmfoundation.org - those are the web sites much more likely to be consulted by someone requiring better accessibility.

**Comment by russbiggs** [comment]
mostly fixed in PR merge 23f40f5
